.github/workflows/set-versions.yml

```diff
@@ -0,0 +1,34 @@
+name: "Set versions"
+on:
```

Comment:
Can this action be run on any branch?

Reply:
I think it can if we add the branch in the checkout action, I could add a choice for master/prod branches I think